code/modules/busy_space/organizations.dm

```diff
@@ -0,0 +1,461 @@
+	// Culture names, because Anewbe told me so.
+	ship_names = list(
+		"Nervous Energy",
+		"Prosthetic Conscience",
+		"Revisionist",
+		"Trade Surplus",
+		"Flexible Demeanour",
+		"Just Read The Instructions",
+		"Limiting Factor",
+		"Cargo Cult",
+		"Gunboat Diplomat",
+		"A Ship With A View",
+		"Cantankerous",
+		"Never Talk To Strangers",
+		"Sacrificial Victim",
+		"Unwitting Accomplice",
+		"Bad For Business",
+		"Just Testing",
+		"Size Isn't Everything",
+		"Yawning Angel",
+		"Liveware Problem",
+		"Very Little Gravitas Indeed",
+		"Zero Gravitas",
+		"Gravitas Free Zone",
+		"Absolutely No You-Know-What",
+		"Existence Is Pain",
+		"Screw Loose",
+		"Of Course I Still Love You",
+		"Limiting Factor",
+		"So Much For Subtley",
+		"Unfortunate Conflict Of Evidence",
+		"Prime Mover",
+		"Boo!",
+		"Reasonable Excuse",
+		"Honest Mistake",
+		"Appeal To Reason",
+		"My First Ship II",
+		"Hidden Income",
+		"Anything Legal Considered",
+		"New Toy",
+		"Me, I'm Always Counting",
+		"Just Five More Minutes",
+		"Are You Feeling It",
+		"Great White Snark",
+		"No Shirt No Shoes",
+		"Callsign"
+	)
+	destination_names = list(
+		"a dockyard on New Canaan"
+	)
+
+/datum/lore/organization/tsc/xion
+	name = "Xion Manufacturing Group"
+	short_name = "Xion"
+	desc = "Xion, quietly, controls most of the market for industrial equipment. Their portfolio includes mining exosuits, \
+	factory equipment, rugged positronic chassis, and other pieces of equipment vital to the function of the economy. Xion \
+	keeps its control of the market by leasing, not selling, their equipment, and through infamous and bloody patent protection \
+	lawsuits. Xion are noted to be a favorite contractor for SolGov engineers, owing to their low cost and rugged design."
+	history = ""
+	work = "industrial equipment manufacturer"
+	headquarters = ""
+	motto = ""
+
+	ship_prefixes = list("XTV" = "hauling")
+	destination_names = list()
+
+// Governments
+
+
+/datum/lore/organization/gov/solgov
+	name = "Solar Confederate Government"
+	short_name = "SolGov"
+	acronym = "SCG"
+	desc = "Colloquially known as SolGov, the TSF is an authoritarian republic that manages the areas in and around the Sol system.\
+			 Despite being a highly militant organization headed by the government of Earth,\
+			 SolGov is usually conservative with its power and mostly serves as a mediator and peacekeeper in galactic affairs."
+	history = "" // Todo
+	work = "governing polity of humanity's Confederation"
+	headquarters = "Luna"
+	motto = "Nil Mortalibus Ardui Est" // Latin, because latin.  Says 'Nothing is too steep for mortals'.
+	autogenerate_destination_names = TRUE
+
+	ship_prefixes = list("FSS-T" = "transportation", "FSS-D" = "diplomatic", "FSS-F" = "freight")
+	destination_names = list(
+		"Venus",
+		"Earth",
+		"Luna",
+		"Mars",
+		"Titan",
+		"Ahdomai",
+		"Kelune",
+		"Dalstadt",
+		"New Canaan",
+		"Jargon 4",
+		"Hoorlm",
+		"Xarxis 5",
+		"Aurum",
+		"Moghes"
```

Comment:
"Haverick"
"Darsing"
"Norfolk"
"Boron 2"
"Iluk"

Also, remove Ahdomai from the list - it's a burnt space rock.